pkg/splunk/enterprise/util.go

```diff
@@ -650,7 +650,7 @@ var resetSymbolicLinks = func(ctx context.Context, client splcommon.ControllerCl
 
 	// Create command for symbolic link creation
 	var command string
-	if crKind == "ClusterManager" {
+	if crKind == "ClusterManager" || crKind == "ClusterMaster" {
 		command = setSymbolicLinkCmanager
```

Comment:
I believe for clusterMaster the location is etc/master-apps while the string points to [setSymbolicLinkCmanager](https://github.com/splunk/splunk-operator/blob/1deadcb2641c9c6d945222963acfcf26e351f436/pkg/splunk/enterprise/names.go#L132). Has the soft link been tested? Maybe I am missing something here.

Reply:
The location is same for Cluster Master as well in any 9.x.x splunk.